resources/styles/components/breadcrumbs/step.less

```diff
@@ -0,0 +1,37 @@
+&-breadcrumbs-step {
+
```

Comment:
Yeah, ok.  I see the intent now.  Hm.  I guess I'm ok with it for a library :grin:   Could you be so kind as to you write a comment on the file explaining it (what you just wrote in the comment is good), so that future me will remember what it's doing ?